code/modules/events/grid_check.dm

```diff
@@ -6,6 +6,10 @@
 
 /datum/event/grid_check/start()
 	power_failure(0)
+	for(var/mob/M in GLOB.player_list)
+		if(!M.client)
+			continue
+		SEND_SOUND(M, sound('sound/effects/powerloss.ogg'))
```

Comment:
Don't do this.

You're creating a sound for every single player in the world, multiple times.

cache it locally and send it to the necessary players. You also shouldn't be sending it to every single player in the world; it should be the same people who can hear the announcement.

Reply:
Maybe a `playsound` `playsound_local` with a fucking immeeeennsseee range that ignores walls and emanates from the engine, wherever it is located on the map

Reply:
okay so what I understand is make a var that has the sound and then send the var?